layouts/partials/site-head.html

```diff
@@ -48,6 +48,11 @@
 {{"<!-- Main Stylesheet -->"|safeHTML}}
 <link rel="stylesheet" href="{{ `output.css` | absURL }}" media="screen">
 
+{{- if .Site.Params.hubspotCode -}}
```

Comment:
Moving this script to the head to see if this may help resolve the cookie banner issue.